006-git.md

```diff
@@ -0,0 +1,25 @@
+# 006 - Git
+
+## Branch workflow
+
+- Create a dedicated branch for each feature. Develop each version in specific branches.
+- Before the first release, main branch is `master`.
+- After the first release, create a new release branch `1.x` and make it main branch.
```

Comment:
Does it worth making it main branch if we have master and it's "latest"?

Reply:
master for major changes, other changes should be added to branch with current major version.